fcrepo-kernel-modeshape/src/test/java/org/fcrepo/integration/kernel/modeshape/FedoraBinaryImplIT.java

```diff
@@ -392,29 +392,29 @@ public void testModifyDatastreamDescriptionLastMod() throws RepositoryException,
             final FedoraBinary orig = binaryService.findOrCreate(session, "/testDatastreamObject/testDatastreamNode6");
             orig.setContent(new ByteArrayInputStream("asdf".getBytes()), "application/octet-stream", null, null, null);
             session.commit();
-            final long origMod = orig.getLastModifiedDate().getTime();
+            final long origMod = orig.getLastModifiedDate().toEpochMilli();
 
             final FedoraResource description = orig.getDescription();
-            final long origDescriptionMod = description.getLastModifiedDate().getTime();
+            final long origDescriptionMod = description.getLastModifiedDate().toEpochMilli();
 
             description.updateProperties(idTranslator, "INSERT { <> <info:fcrepo/foo> \"b\" } WHERE {}",
                     description.getTriples(idTranslator, PROPERTIES));
             session.commit();
 
             final FedoraBinary ds = binaryService.findOrCreate(session, "/testDatastreamObject/testDatastreamNode6");
 
-            final long modMod = ds.getLastModifiedDate().getTime();
-            final long modDescMod = ds.getDescription().getLastModifiedDate().getTime();
+            final long modMod = ds.getLastModifiedDate().toEpochMilli();
+            final long modDescMod = ds.getDescription().getLastModifiedDate().toEpochMilli();
```

Comment:
do these `Instant` values need to be converted to `long`s? Why not just keep them as `Instant`s?